### ISSUE TITLE
FIRE has extra space on first line of code blocks

### ISSUE THREAD

**Issue by tripleee** [Charcoal-SE/userscripts#144]
It seems that code formatted with four spaces gets indented an extra space on the first line.

Sample screenshot from Chrome on MacOS Mojave, input is https://metasmoke.erwaysoftware.com/post/166877

![Screen shot illustrating problem](https://user-images.githubusercontent.com/2160915/55006091-d61f6e80-4fe5-11e9-9669-b11ee96f4e6c.png)

I originally though this was because of the recent code formatting update in Stack Overflow's Markdown (and maybe it is) but this seems to affect the traditional code markup.

**Comment by double-beep** [comment]
This is actually how the code **is** in the SO post, where is the problem?

**Comment by Undo1** [comment]
@double-beep Are you sure? I'm not seeing it on [the actual post](https://stackoverflow.com/posts/55359097/revisions).

**Comment by CalvT** [comment]
Actually @double-beep  is correct. See below - note the blue selected area where I have selected the spaces:

![image](https://user-images.githubusercontent.com/15909971/55009563-85961a00-4fc1-11e9-9929-fcec07b72130.png)

The spaces are in the original post, so not a FIRE problem.

**Comment by CalvT** [comment]
The extra space is present on all code blocks, be it ````, ~~~, or the standard four spaces:

Examples - code with `~~~`

![image](https://user-images.githubusercontent.com/15909971/55010314-bf1b5500-4fc2-11e9-8a60-93b339ef4cca.png)

Code indented with four spaces:

![image](https://user-images.githubusercontent.com/15909971/55010446-effb8a00-4fc2-11e9-8a02-13683b3521c9.png)

Reopening with a title change.

**Comment by makyen** [comment]
This is a CSS issue with an easy fix. I'll push an update. The problem is that the 

```
.fire-popup .fire-popup-body code {
    padding: 1px 5px;
}
```
has strange effects when it's within a `<pre>`. That's easily solved.

**Comment by makyen** [comment]
Resolved/fixed with commit [dd69464](https://github.com/Charcoal-SE/userscripts/commit/dd69464c7dc02ffe263aaf6d35c2c7a494338915).
